### ISSUE TITLE
Event callbacks not working at all

### ISSUE THREAD

**Issue by saidarshan27** [chintan9/plyr-react#1059]
**Describe the bug**
Event callbacks like onPlay, onClick, onPause not working at all. I tried to reproduce it in a codesandbox.

```
import "./styles.css";
import Plyr from "plyr-react";
import "plyr-react/plyr.css";

export default function App() {
  return (
    <Plyr
      type="youtube"
      videoId="umldliRDefQ"
      onPlay={() => console.log("playing")}
      onClick={() => {
        console.log("clicking");
      }}
    />
  );
}
```

**To Reproduce**
1. Go to [(https://codesandbox.io/s/recursing-morning-b1mk1t?file=/src/App.js](demo)

**Expected behavior**
I want a callback to be run onPlay, onPause, onSeek to attach custom handlers.

[https://stackoverflow.com/questions/76065265/plyr-react-event-callbacks-not-working-at-all](stackoverflow question)

**Comment by saidarshan27** [comment]
nothing

**Comment by metacoding** [comment]
Yes I confirm this issue exists, a few handlers work others don't

**Comment by sloreti** [comment]
Look at the `listeners` attribute under the `options` prop. 

Docs: https://github.com/sampotts/plyr#options
Available event handlers: https://github.com/sampotts/plyr/blob/a148e2e5b6eb4cda1d63a21397636a365f5dabbb/src/js/config/defaults.js#L214
